lib/ansible/plugins/netconf/__init__.py

```diff
@@ -223,9 +223,9 @@ def dispatch(self, rpc_command=None, source=None, filter=None):
         """
         if rpc_command is None:
             raise ValueError('rpc_command value must be provided')
-        req = fromstring(rpc_command)
-        resp = self.m.dispatch(req, source=source, filter=filter)
-        return resp.data_xml if resp.data_ele else resp.xml
+
```

Comment:
We recently found this breaks on iosxr, so we need to hold off on this backport until we fix this on devel